test/form/index.js

```diff
@@ -47,7 +47,8 @@ runTestSuiteWithSamples('form', path.resolve(__dirname, 'samples'), (dir, config
 							{
 								exports: 'auto',
 								file: inputFile,
-								format: defaultFormat
+								format: defaultFormat,
+								// validate: true // add when systemjs bugs fixed
```

Comment:
Commented out to pass tests.
Enable validation here to see the issues related to:
```
test/form/samples/system-export-compact/_expected.js
test/form/samples/system-multiple-export-bindings/_expected.js
test/form/samples/no-external-live-bindings-compact/_expected/system.js
```